rasa/cli/test.py

```diff
@@ -115,29 +116,51 @@ def run_core_test(args: argparse.Namespace) -> None:
     )
 
 
-def run_nlu_test(args: argparse.Namespace) -> None:
-    """Run NLU tests."""
+async def run_nlu_test_async(
+    config: Optional[Union[Text, Dict]],
+    data_path: Optional[Text],
+    models_path: Optional[Text],
+    output_dir: Optional[Text],
+    cross_validation: Optional[bool],
+    percentages: Optional[List[int]],
+    runs: Optional[int],
+    no_errors: Optional[bool],
+    all_args: Optional[Dict[Text, Any]],
+) -> None:
+    """Runs NLU tests.
+
+    Args:
+        all_args: all arguments gathered in a Dict so we can pass them to 'perform_nlu_cross_validation' and
+                  'test_nlu' as parameter.
+        config: config file or a list of multiple config files.
+        data_path: path for the nlu data.
+        models_path: the path for the nlu data.
+        output_dir: the directory for the results to be saved.
+        cross_validation: boolean value that indicates if it should test the model using cross validation or not.
```

Comment:
I suggest not repeating the types in docstrings as we have the type annotations for this.
```suggestion
        cross_validation: indicates if it should test the model using cross validation or not.
```

Reply:
True! I removed all types from doc strings.

Reply:
slightly related and nice to know: https://en.wikipedia.org/wiki/Hungarian_notation